src/Handler.php

```diff
@@ -301,6 +306,7 @@ protected function getOptions() {
     $options = $extra['drupal-l10n'] + [
       'destination' => 'sites/default/files/translations',
       'languages' => [],
+      'core_version' => '',
```

Comment:
Documentation in README.md needs to be updated and tests to cover this feature needs to be written.